console/src/range/overview/Details.tsx

```diff
@@ -186,82 +185,84 @@ export const Details: FC<DetailsProps> = ({ rangeKey }) => {
       // Retrieve ${name}
       const rng = await client.ranges.retrieve("${rangeKey}")
     `,
-      `TypeScript code for ${name}`,
+      `TypeScript code to retrieve ${name}`,
     );
   };
 
   return (
     <Form.Form {...formCtx}>
-      <Align.Space direction="x" justify="spaceBetween" className={CSS.B("header")}>
-        <Align.Space direction="y" grow>
-          <Form.TextField
-            path="name"
-            showLabel={false}
-            inputProps={{
-              variant: "natural",
-              level: "h1",
-              placeholder: "Name",
-              onlyChangeOnBlur: true,
-              resetOnBlurIfEmpty: true,
-            }}
-            padHelpText={false}
-          />
-          <ParentRangeButton rangeKey={rangeKey} />
-        </Align.Space>
-        <Align.Space
-          direction="x"
-          className={CSS.B("copy-buttons")}
-          style={{ height: "fit-content" }}
-          size="small"
-        >
-          <Align.Space direction="x">
-            <Button.Icon
-              tooltip={`Copy Python code for ${name}`}
-              tooltipLocation="bottom"
-              variant="text"
-            >
-              <Icon.Python
-                onClick={handleCopyPythonCode}
-                style={{ color: "var(--pluto-gray-l7)" }}
-              />
-            </Button.Icon>
+      <Align.Space direction="y" size="large">
+        <Align.Space direction="x" justify="spaceBetween" className={CSS.B("header")}>
+          <Align.Space direction="y" grow>
+            <Form.TextField
+              path="name"
+              showLabel={false}
+              inputProps={{
+                variant: "natural",
+                level: "h1",
+                placeholder: "Name",
+                onlyChangeOnBlur: true,
+                resetOnBlurIfEmpty: true,
+              }}
+              padHelpText={false}
+            />
+            <ParentRangeButton rangeKey={rangeKey} />
+          </Align.Space>
+          <Align.Space
+            direction="x"
+            className={CSS.B("copy-buttons")}
+            style={{ height: "fit-content" }}
+            size="small"
+          >
+            <Align.Space direction="x">
+              <Button.Icon
+                tooltip={`Copy Python to retrieve ${name}`}
```

Comment:
I think the tooltips as "Copy Python code to retrieve ${name}" is better, but if you think its too verbose we can keep it like this.